test/rpc_test.py

```diff
@@ -836,6 +855,12 @@ def test_rref_forward_chain(self):
         ret = ret_rref
         self.assertEqual(ret, torch.add(torch.ones(n, n), 1))
 
+    @dist_init
+    @requires_process_group_agent("PROCESS_GROUP rpc backend specific test, skip")
+    def test_timeout_set(self):
+        rpc_timeout_seconds = rpc.get_rpc_timeout().seconds
+        self.assertEqual(rpc_timeout_seconds, 100)
```

Comment:
I have the default timeout for all RPCs to be 100 seconds for now, but I'm not sure what a good default is. If it's too small, users could get time outs when in fact the operation was going to complete successfully.

Reply:
Let's use 10s for now but also provide a way to specify infinite timeout. Some RPC frameworks use timeout=0 for this. We should document this as well.